### ISSUE TITLE
feat: Adding a searchbar for the settings inside the RevancedExtended settings

### ISSUE THREAD

**Issue by ArcherLegend** [anddea/revanced-patches#68]
### Application

YouTube

### Feature description

There should be a searchbar at the top of the settings section inside the Youtube Revanced Extended settings page in order to be able to quickly search for the settings you need to change

### Motivation

It will make like a lot easier when you want to look for a setting.

### Acknowledgements

- [X] This issue is not a duplicate of an existing feature request.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] I have written the title and contents in English.

**Comment by Francesco146** [comment]
after #397, i'd like to take a look at this

**Comment by ArcherLegend** [comment]
That is good, I hope that you'll have a look at #111 
Il fatto di poter trascinare in un'area dello schermo che attiva solo il controllo volume o audio sarebbe fantastico perchè molte volte si attiva invece lo scrubbing sulla timeline.
